neuralmonkey/train.py

```diff
@@ -48,7 +46,7 @@ def create_config() -> Configuration:
     config.add_argument('postprocess')
     config.add_argument('name', str)
     config.add_argument('random_seed', int, required=False)
-    config.add_argument('initial_variables', Union[List[str], str],
```

Comment:
Stěžovalo si mi na to mypy, že je to za runtimu k ničemu, tak jsem to dal pryč. Ale stejně tam tyhle typy jsou zbytečně, protože se o to už stará typeguard v training loop.